### PR TITLE
apt-get: add Farsi translation

### PR DESCRIPTION
**Pull request by FarzadHayat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** N/A

### CODE REVIEW

File: pages.fa/linux/apt-get.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`apt-get install {{package}}`
+
+- حذف یک بسته:
+
+`apt-get remove {{package}}`
+
+- حذف یک بسته و فایل‌های پیکربندی آن:
+
+`apt-get purge {{package}}`
```
Comment (kbdharun) [inline pages.fa/linux/apt-get.md:21]:
```suggestion
- نصب یک بسته یا به روزرسانی آن به آخرین نسخه موجود:

`apt-get install {{نام کاربری}}`

- حذف یک بسته:

`apt-get remove {{نام کاربری}}`

- حذف یک بسته و فایل‌های پیکربندی آن:

`apt-get purge {{نام کاربری}}`
```

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, we generally translate the placeholders to facilitate a better understanding of examples for native readers.

File: pages.fa/linux/apt-get.md (hunk 1 above)
Comment (FarzadHayat) [reply pages.fa/linux/apt-get.md]: Thanks for letting me know!
The correct term for package would be "بسته".
I have pushed a commit with the changes.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**